metrics/web/test/common/presentation/tooltip_icon/widgets/tooltip_icon_test.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/base/presentation/widgets/svg_image.dart';
+import 'package:metrics/common/presentation/metrics_theme/model/metrics_theme_data.dart';
+import 'package:metrics/common/presentation/tooltip_icon/theme/theme_data/tooltip_icon_theme_data.dart';
+import 'package:metrics/common/presentation/tooltip_icon/widgets/tooltip_icon.dart';
+import 'package:metrics/common/presentation/tooltip_popup/widgets/tooltip_popup.dart';
+import 'package:network_image_mock/network_image_mock.dart';
+
+import '../../../../test_utils/finder_util.dart';
+import '../../../../test_utils/metrics_themed_testbed.dart';
+
+void main() {
+  group("TooltipIcon", () {
+    const themeData = MetricsThemeData(
+      tooltipIconTheme: TooltipIconThemeData(
+        color: Colors.red,
+        hoverColor: Colors.yellow,
+      ),
+    );
+
+    final mouseRegionFinder = find.ancestor(
+      of: find.byType(SvgImage),
+      matching: find.byType(MouseRegion),
+    );
+
+    Future<void> _hoverBar(WidgetTester tester) async {
+      final mouseRegion = tester.widget<MouseRegion>(mouseRegionFinder);
+      mouseRegion.onEnter(const PointerEnterEvent());
+      await mockNetworkImagesFor(() {
+        return tester.pumpAndSettle();
+      });
```

Comment:
It seems like there is no need in `mockNetworkImagesFor` here. 
```suggestion
      return tester.pumpAndSettle();
```